redash/query_runner/mysql.py

```diff
@@ -99,11 +99,11 @@ def enabled(cls):
 
     def _get_tables(self, schema):
         query = """
-        SELECT col.table_schema,
-               col.table_name,
-               col.column_name
+        SELECT col.TABLE_SCHEMA as table_schema,
+               col.TABLE_NAME as table_name,
+               col.COLUMN_NAME as column_name
```

Comment:
Isn't this breaking for anyone who didn't set `local_case_table_names` to 0?

Reply:
I'll answer instead: no, it will not, because originally all column names in `information_schema` are upper-cased. When `lower_case_table_names == 0` - then query must use upper-cased names as well; when `lower_case_table_names != 0` - no matter what case the query use, MySQL will use CI comparison. So common denominator is to always use upper-cased names.